src/actions/index.js

```diff
@@ -1,8 +1,20 @@
 // we'll need axios
-
+import axios from "axios";
 // we'll need to create 3 different action types here.
 // one for fetching, one for success and one for failure
-
+export const FETCHING_DATA = "FETCHING_DATA";
+export const FETCHING_SUCCESS = "FETCHING_SUCCESS";
+export const FETCHING_ERROR = "FETCHING_ERROR";
```

Comment:
Instead of making your action types name general I'd provide more context in them.

_e.g_ 
instead of export const FETCHING_DATA = "FETCHING_DATA"; 
I would do export const FETCH_STARWARS_CHARS_DATA= "FETCH_STARWARS_CHARS_DATA";

It may be more verbose but it's more descriptive which can help a lot in bigger applications.